modules/agones/main.tf

```diff
@@ -83,7 +83,9 @@ module "agones_system_node_group" {
   min_size               = 1
   max_size               = 10
   desired_size           = 1
-  instance_types         = ["t3.large"]
+  # ami_type               = "AL2_ARM_64"
```

Comment:
`agones-controller`  has bugs in ARM, and will fix Agones 1.24.0
https://github.com/googleforgames/agones/issues/2578